piezo_web_app/PiezoWebApp/tests/handlers/get_jobs_test.py

```diff
@@ -0,0 +1,65 @@
+from mock import call
+from tornado.testing import gen_test
+
+from PiezoWebApp.tests.handlers.base_handler_test import BaseHandlerTest
+from PiezoWebApp.src.handlers.get_jobs import GetJobsHandler
+
+
+class TestGetLogsHandler(BaseHandlerTest):
```

Comment:
Class name is wrong

Reply:
> Looks good, but the [Web App User Guide](https://github.com/ukaea/piezo/wiki/WebAppUserGuide) also needs updating

https://github.com/ukaea/piezo/wiki/WebAppUserGuide#get-list-of-jobs

Reply:
f298795